hdinsight-linux-with-hue-on-edge-node/azuredeploy.json

```diff
@@ -0,0 +1,156 @@
+            "metadata": { 
+                "description": "The name of the edge node virtual machine where Hue app is installed."
+            }
+        },
+        "edgeNodeVirtualMachineSize": {
+            "type": "string",
+            "defaultValue": "Standard_D1",
+             "allowedValues": [
+                "Standard_A1",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4",
+                "Standard_D1",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4"
+              ],
+            "metadata": { 
+                "description": "The size of the edge node virtual machine."
+            }
+        }
+    },
+    "variables": {
+        "defaultApiVersion": "2015-01-01"
+    },
+    "resources": [
+        {
+            "name": "HDInsightDeployment",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "[variables('defaultApiVersion')]",
+            "dependsOn": [
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/ezubatov/azure-quickstart-templates/master/hdinsight-linux-with-edge-node/azuredeploy.json",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "location": { "value": "[parameters('location')]" },
+                    "clusterVNetAddressSpace": { "value": "[parameters('clusterVNetAddressSpace')]" },
+                    "clusterVNetSubnetName": { "value": "[parameters('clusterVNetSubnetName')]" },
+                    "clusterVNetSubnetAddressRange": { "value": "[parameters('clusterVNetSubnetAddressRange')]" },
+                    "clusterName": { "value": "[parameters('clusterName')]" },
+                    "clusterLoginUserName": { "value": "[parameters('clusterLoginUserName')]" },
+                    "clusterLoginPassword": { "value": "[parameters('clusterLoginPassword')]" },
+                    "sshUserName": { "value": "[parameters('sshUserName')]" },
+                    "sshPassword": { "value": "[parameters('sshPassword')]" },
+                    "clusterStorageAccountName": { "value": "[parameters('clusterStorageAccountName')]" },
+                    "clusterWorkerNodeCount": { "value": "[parameters('clusterWorkerNodeCount')]" },
+                    "edgeNodeVhdStorageAccountName": { "value": "[parameters('edgeNodeVhdStorageAccountName')]" },
+                    "edgeNodeVirtualMachineName": { "value": "[parameters('edgeNodeVirtualMachineName')]" },
+                    "edgeNodeVirtualMachineSize": { "value": "[parameters('edgeNodeVirtualMachineSize')]" },
+                    "edgeNodeSetupScriptUri": { "value": "https://raw.githubusercontent.com/ezubatov/azure-quickstart-templates/master/hdinsight-linux-with-hue-on-edge-node/install-hue-edge-v01.sh" }
```

Comment:
This should also point to the azure repository. Just replace ezubatov with Azure